tools/prune_database.sh

```diff
@@ -8,16 +8,22 @@
 source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
 
 # Check if local postgres server is running
-if nc -w1 localhost "${INTEGREAT_CMS_DB_PORT}"; then
+if nc -z -w1 localhost "${INTEGREAT_CMS_DB_PORT}"; then
```

Comment:
With docker, the `z` flag did not make a difference in my testting, but ith psql, the absence of `-z` led to `nc` never exiting.